system/Router/Router.php

```diff
@@ -330,7 +330,7 @@ protected function checkRoutes(string $uri): bool
 
         $uri = $uri === '/'
```

Comment:
Can we make this a one-liner?

Reply:
How do you mean one-liner it is already one line ?
 
```
        $uri = $uri === '/'
            ? $uri
            : trim($uri, '/ ');
```

Maybe

```
$url = trim($url) ?: '/';
```

Reply:
Yes.
```php
$uri = trim($uri, '/ ') ?: '/';
```